atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -998,7 +1029,12 @@ private static LockAndTimestampServices getLockAndTimestampServices(
                 RemoteLockServiceAdapter.create(lockRpcClient, timelockNamespace));
 
         ConjureTimelockService conjureTimelockService = new BlockingSensitiveConjureTimelockService(
-                BlockingAndNonBlockingServices.create(creator, ConjureTimelockService.class));
+                ImmutableBlockingAndNonBlockingServices.<ConjureTimelockService>builder()
+                        .nonBlocking(clientFactory.client(ConjureTimelockService.class))
```

Comment:
this means we'd end up using the feign shim right? is the plan still to get to the full zero-reflection codegen version? (presumably would need to open-source carter's ConjureServiceAdapter proxy thingy.)

Reply:
Ya originally the plan was to go straight to full dialogue, but I wanted to avoid copying all of carter's proxy so I went for the shim as a starting point